ttn/discovery.py

```diff
@@ -19,11 +19,10 @@
 
 class DiscoveryClient:
 
-    def __init__(self, discovery_address=None, certificate=None):
+    def __init__(self,
+                 discovery_address="discovery.thethings.network:1900",
+                 certificate=None):
```

Comment:
Type consistency: Use `""` for an empty string rather than `None`. With `""`, the app won't crash if you're calling `str` methods, and you still have access to built-in [value testing grammar](https://docs.python.org/3.7/library/stdtypes.html):

```python
if certificate:
    self.certificate = certificate
```

```python
if not certificate:
    print("No certificate was passed")
```